src/helpers/quoteAPI.js

```diff
@@ -0,0 +1,10 @@
+const quoteAPI = async () => {
+  const response = await fetch('https://random-math-quote-api.herokuapp.com/');
+  const json = await response.json();
+  return {
+    author: json.author,
+    content: json.quote,
+  };
+};
+
+export default quoteAPI;
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/199648059-c1700180-297d-43e2-8d2b-825b321a532e.png)

- [ ] You can see that all the tests are not passing. Kindly make your application work as expected so all the tests will pass :blush:

Reply:
Hello, @HAMMAS-SALEEM I think it is a misunderstanding, I've sent you a message on slack, and the test is passing, it could be the API response taking long, I'll send you the screenshot with all tests passing, and also with the code so you can see there's no modification:
![image](https://user-images.githubusercontent.com/79666590/199650235-7727b36a-ea77-4940-81ba-0b6ae1eb9564.png)

Reply:
well resolved! ✔️

Reply:
Hey @OscarFMdev,
That's great you have addressed the issue. You are good to go my dear brother 😊